PocketKit/Sources/PocketKit/Home/HomeCollectionViewSectionProvider.swift

```diff
@@ -124,3 +124,27 @@ class HomeViewControllerSectionProvider {
         return section
     }
 }
+
+
+extension HomeViewControllerSectionProvider {
+    func offlineSection(environment: NSCollectionLayoutEnvironment, withRecentSaves: Bool) -> NSCollectionLayoutSection {
+        var config = UICollectionLayoutListConfiguration(appearance: .plain)
+        config.backgroundColor = UIColor(.ui.white1)
+        config.showsSeparators = false
+        let section = NSCollectionLayoutSection.list(using: config, layoutEnvironment: environment)
+
+        let layoutHeight = environment.container.contentSize.height
+        let availableWidth = environment.container.contentSize.width
+        - ItemsListOfflineCell.Constants.padding
+        - ItemsListOfflineCell.Constants.padding
+        let topInset: CGFloat = withRecentSaves ? 16 : (layoutHeight - ItemsListOfflineCell.height(fitting: availableWidth)) / 2 - 128
```

Comment:
interested to understand why we are diving by 2 and subtracting 128. 🤔  also the offline view seems to be a bit cut off on smaller devices

![IMG_1AB1E29CC542-1](https://user-images.githubusercontent.com/6743397/185412881-caabb1b8-16aa-48e1-bf9c-e223dfb67845.jpeg)

Reply:
@cyndichin good call-out with the "magic numbers"; i think this is a place I can refactor (and hopefully fix getting cut off)

Reply:
looks great now!